### PR TITLE
T#296 Add `teams edit` command

### PR DESCRIPTION
**Pull request by simenheg**
Add `teams edit` command for editing team details.

~~Depends on https://github.com/oslokommune/okdata-permission-api/pull/58, https://github.com/oslokommune/okdata-cli/pull/170, and https://github.com/oslokommune/okdata-sdk-python/pull/102 being released (and `setup.py` + `requirements.txt` being updated in this PR to reflect this).~~

### CODE REVIEW

File: okdata/cli/commands/teams/wizards.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    q_attribute_value,
+    q_team,
+    q_team_name,
+)
+from okdata.cli.commands.wizard import run_questionnaire
+
+
+def edit_team_wizard(team_client):
+    choices = run_questionnaire(
+        q_team(team_client),
```
Comment (petterhj) [inline okdata/cli/commands/teams/wizards.py:12]: Trenger denne en `my=True`?

Comment (simenheg) [reply okdata/cli/commands/teams/wizards.py:12]: `include=None` er default i SDK-klienten.

Comment (simenheg) [reply okdata/cli/commands/teams/wizards.py:12]: Ah, sorry, nå skjønte jeg hva du mente. Ja, den må få `my=True` når den parameteren legges til i https://github.com/oslokommune/okdata-cli/pull/173.

**Review by petterhj — APPROVED** [review]
LGTM! <3

**Pull request merged**